### PR TITLE
docs: Add Helm extension docs

### PR DESCRIPTION
**Pull request by cabrinha**
Merge after this:
- https://github.com/zed-industries/extensions/pull/746

Release Notes:

- N/A

### CODE REVIEW

File: docs/src/languages/helm.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "Helm": [
+      "**/templates/**/*.tpl",
+      "**/templates/**/*.yaml",
+      "**/templates/**/*.yml",
+      "**/helmfile.d/**/*.yaml"
+    ]
+  }
+```
+
+- Language Server: [tree-sitter-go-template](https://github.com/ngalaiko/tree-sitter-go-template/tree/master)
```
Comment (qvalentin) [inline docs/src/languages/helm.md:20]: This is not the lsp, but just the tree-sitter grammer, both should be linked

Comment (qvalentin) [reply docs/src/languages/helm.md:20]:
```markdown
- tree-sitter grammar: [tree-sitter-go-template](https://github.com/ngalaiko/tree-sitter-go-template)
- Language Server: [helm-ls](https://github.com/mrjosh/helm-ls/)
```

File: docs/src/languages/helm.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Helm
+
+Support for `helm` in Zed is provided by the community-maintained [Helm extension](https://github.com/cabrinha/helm.zed).
+
+- Tree-sitter: [tree-sitter-go-template](https://github.com/ngalaiko/tree-sitter-go-template/tree/master)
+- Language Server: [mrjosh/helm-ls](https://github.com/mrjosh/helm-ls)
+
+## Setup
+
+Enable the helm filetypes by editing you `.zed/settings.json` and adding:
```
Comment (hedgieinsocks) [inline docs/src/languages/helm.md:10]: you -> your

File: docs/src/languages/helm.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+```json
+  "file_types": {
+    "Helm": [
+      "**/templates/**/*.tpl",
+      "**/templates/**/*.yaml",
+      "**/templates/**/*.yml",
+      "**/helmfile.d/**/*.yaml"
```
Comment (hedgieinsocks) [inline docs/src/languages/helm.md:18]: perhaps add the same line for .yml

**Review by maxdeviant — APPROVED** [review]
Thanks!

**Pull request merged**